cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -483,39 +488,56 @@ construct_runtime!(
 		Utility: pallet_utility::{Pallet, Call, Event} = 40,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 36,
 
-		// BridgeHubRococo uses:
-		//  - BridgeWestendGrandpa
-		//  - BridgeWestendParachains
-		//  - BridgeWestendMessages
-		//  - BridgeRelayers
+		// Bridge relayers pallet, used by several bridges here.
```

Comment:
I've changed the grouping here - this runtime is no longer shared by Rococo and Wococo, so imo it makes sense to group by bridged chain.